src/org/opendatakit/validate/FormValidator.java

```diff
@@ -123,11 +123,21 @@
     public static void main(String[] args) {
         try {
             UIManager.setLookAndFeel(UIManager.getSystemLookAndFeelClassName());
-            if (args.length == 1) {
-                String path = args[0];
-                new FormValidator().validateAndExitWithErrorCode(path);
-            } else {
+            if (args.length == 0) {
                 new FormValidator().show();
+            } else {
+                List<String> paths = new ArrayList<>();
+                boolean failFast = false;
+                for (String arg : args) {
+                    if (arg.equals("--failFast") || arg.equals("--fail-fast")) {
```

Comment:
We have to remember documenting the new argument in the docs! :)

Could you add links in the PR description to the places in the docs/readmes where we should make changes after this one gets merged, please?

Reply:
I believe opendatakit/docs#1045 should be the only place?

Reply:
I think it's enough and I'm sure someone will complain if it's not :D